5-Clustering/2-K-Means/README.md

```diff
@@ -242,7 +242,7 @@ Hint: Try to scale your data. There's commented code in the notebook that adds s
 
 ## Review & Self Study
 
-Take a look at Stanford's K-Means Simulator [here](https://stanford.edu/class/engr108/visualizations/kmeans/kmeans.html). You can use this tool to visualize sample data points and determine its centroids. With fresh data, click 'update' to see how long it takes to find convergence. You can edit the data's randomness, numbers of clusters and numbers of centroids. Does this help you get an idea of how the data can be grouped?
+Take a look at K-Means Simulator [here](https://user.ceng.metu.edu.tr/~akifakkus/courses/ceng574/k-means/). You can use this tool to visualize sample data points and determine its centroids. You can edit the data's randomness, numbers of clusters and numbers of centroids. Does this help you get an idea of how the data can be grouped?
```

Comment:
Can you edit this to "Take a look at a K-Means Simulator [such as this one](link)"

Reply:
I'll fix it, thanks

Reply:
Thanks for the review Jen. I am changing this to the text you mentioned.